orca/orca-clouddriver/orca-clouddriver.gradle

```diff
@@ -32,12 +32,14 @@ dependencies {
   implementation(project(":orca-deploymentmonitor"))
   implementation("io.spinnaker.kork:kork-moniker")
   implementation("io.spinnaker.kork:kork-retrofit")
+  implementation("io.spinnaker.kork:kork-retrofit2")
```

Comment:
Why this dependency?  I see Retrofit2ServiceFactory used in e.g. CloudDriverConfigurationTest, so I can see a testImplementation dependency.  Does the code in main need it to, e.g. for Retrofit2ServiceFactoryAutoConfiguration ?